frontend/src/alert_configuration/alert_configuration.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+//import Album from './Album';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+serviceWorker.unregister();
```

Comment:
what is `serviceWorker` used for? if it came from the create-react-app we probably won't need it for our web app